### ISSUE TITLE
Can't change icon button of the upload plugin

### ISSUE THREAD

**Issue by SadegOussama** [Alex-D/Trumbowyg#974]
### Description
I tried to modify the upload button icon, but unfortunately the plugin does not offer this feature. Could you make this possible ?

For the moment I don't have a workaround solution, if you have one I'm a buyer.

### How to reproduce?
  - Add the upload plugin and the upload button.
  - In the btnsDef, change the icon to another one (ex: 'insert-image').
  - Refresh the page and look the result.

Thank you.

**Comment by Alex-D** [comment]
Ah! You right... 
Should be fixed in the 2.18.0 late this day :)

You can do that:

```js
$('#editor').trumbowyg({
    btns: ['upload'],
    btnsDef: {
      upload: {
        ico: 'insertImage'
      }
    }
});
```

**Comment by SadegOussama** [comment]
Thank you for your reactivity!
